ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -261,7 +300,8 @@ public void setupResources(final Configurable<?> config, final KsqlRestConfig ap
 
   @Override
   public void startAsync() {
-    log.info("KSQL RESTful API listening on {}", StringUtils.join(getListeners(), ", "));
+    startApiServer(ksqlConfigNoPort);
```

Comment:
Is it important that this call is made before the REST server is started? (Wondering if there's a reason this call was moved to the top.)

Reply:
Yep, the apiServer now serves al traffic and has the real listeners so needs to be started first.